articles/blockchain-as-a-revolution-inhealth/index.md

```diff
@@ -0,0 +1,86 @@
+---
+ layout: engineering-education
+ status: publish
+ published: true
+ url: /engineering-education/blockchain-as-a-revolution-in-health/
+ title: Blockchain as a Revolution in Health
+ description: Learn how Blockchain technology can provide a greater value of care and better health outcomes for patients.
+ author: jethro-magaji
+ date: 2020-08-24T00:00:00-10:00
+ topics: [Blockchain][Cryptocurrency][Healthtech]
+ excerpt_separator: <!--more-->
+ images:
+
+   - url: /engineering-education/blockchain-as-a-revolution-inhealthcare/Blockchain-Health-img.jpg
+     alt: Blockchain Health
+ ---
+ 
+[*Image Source: https://www.gettyimages.com/detail/illustration/caduceus-icon-on-money-and-cryptocurrency-royalty-free-illustration/920245220*](https://www.gettyimages.com/detail/illustration/caduceus-icon-on-money-and-cryptocurrency-royalty-free-illustration/920245220)
+
+# Blockchain as a Revolution in Healthcare
+![Blockchain img](/engineering-education/blockchain-as-a-revolution-inhealthcare/Blockchain-img.jpg)
+[*Image Source: https://www.gettyimages.com/detail/illustration/digital-room-with-padlock-and-word-royalty-free-illustration/748345923?adppopup=true*](https://www.gettyimages.com/detail/illustration/digital-room-with-padlock-and-word-royalty-free-illustration/748345923)
+
+[Blockchain represents great promise for stakeholders in healthcare fields](https://nimbus-t.com/blockchain-opportunities-for-health-care-deloitte-us). Using this technology has the potential to increase the standard of care. In the long run, using a [blockchain network nationwide](https://www.parathon.com/understanding-blockchain-in-healthcare-and-how-it-is-beneficial-to-providers.html) for electronic medical records (EMR) could help achieve better health outcomes for patients and improve efficiencies in healthcare delivery.
```

Comment:
Complete the sentence"Using this technology has the potential to increase the standard of care because ..." to explain why Blockchain has this potential. Feel free to summarize your points below.

Reply:
Done